images/linux/toolsets/toolset-1804.json

```diff
@@ -123,13 +123,14 @@
             "name": "az",
             "url" : "https://raw.githubusercontent.com/Azure/az-ps-module-versions/main/versions-manifest.json",
             "versions": [
-                "7.5.0"
+                "9.0.1"
```

Comment:
Maybe we shouldn't update if for Ubuntu 18.04 since we are deprecating it soon

Reply:
it is 3 and a half months ahead, nobody knows if we will have a need of releasing ubuntu once more, so I'd keep all in sync